keyboards/cannonkeys/an_c/config.h

```diff
@@ -62,21 +62,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // EEPROM usage
 // TODO: refactor with new user EEPROM code (coming soon)
 #define EEPROM_MAGIC 0x451F
-#define EEPROM_MAGIC_ADDR 32
+#define EEPROM_MAGIC_ADDR 40
 // Bump this every time we change what we store
 // This will automatically reset the EEPROM with defaults
 // and avoid loading invalid data from the EEPROM
 #define EEPROM_VERSION 0x02
```

Comment:
One thing that was mentioned on discord, was that `EEPROM_VERSION` had to be updated to fix boards flashed with previous eeprom conflict firmware.

Reply:
Fixed! Thanks for letting me know